util/src/main/java/com/google/cloud/hadoop/util/HttpTransportFactory.java

```diff
@@ -42,10 +48,14 @@
    * @return The resulting HttpTransport.
    * @throws IllegalArgumentException If the proxy address is invalid.
    * @throws IOException If there is an issue connecting to Google's Certification server.
+   * @param socketKeepAlive Whether to keep TCP socket alive
    */
-  public static HttpTransport createHttpTransport() throws IOException {
+  public static HttpTransport createHttpTransport(boolean socketKeepAlive) throws IOException {
```

Comment:
Instead of breaking change, can we introduce a separate method ?

Reply:
Sure, I have kept the method signature same and no longer need to pass this param as it is only being called with `true`.

Reply:
If we will not customize this setting in a first place, we will not need this change at all.

Reply:
Not allowing to configure now.